redirect-robots.php

```diff
@@ -278,18 +278,14 @@ function deny( ) {
 }
 
 /**
- * check for preview & verify
+ * check for staging machines (for example: preview or verify)
  */
 $headers = function_exists('apache_request_headers') ? apache_request_headers() : array();
 
-$isdeny = !empty( $headers[ "X-Staging" ] ) &&
-	preg_match("@^(sandbox|preview$|verify$)@", $headers["X-Staging"])
-	? true
-	: false;
+$isdeny = !empty( $headers[ "X-Staging" ] ) ? true : false;
```

Comment:
Of course you're right @frankfarmer! Thanks for the hint and link to similar discussion :)